components/Uploader/Uploader.js

```diff
@@ -51,6 +51,32 @@ const Error = styled.div`
   color: ${({ theme }) => theme.red};
 `;
 
+const splitFilesToUniqAndDuplicates = files => {
+  const uniqFiles = [];
+  const duplicatedFiles = [];
+
+  const map = new Map();
+
+  // eslint-disable-next-line
+  for (const file of files) {
+    const checkLabel = `${file.name}_${file.raw.length}`;
+
+    if (!map.has(checkLabel)) {
+      map.set(checkLabel, true);
+      uniqFiles.push(file);
+    } else {
+      duplicatedFiles.push(file);
+    }
+  }
+
+  return { uniqFiles, duplicatedFiles };
+};
+
+const getNotificationText = files =>
+  files.length > 1
+    ? `${files.map(({ name }) => name).join(', ')} have already added`
+    : `The ${files[0].name} has already added`;
```

Comment:
`The ${files[0].name} has already added` => `The file with name "${files[0].name}" has already added`